roles/openshift_repos/vars/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
 known_openshift_deployment_types: ['origin', 'online', 'enterprise']
+known_openshift_product_types: ['openshift', 'atomic-enterprise']
```

Comment:
@ashcrow @aveshagarwal @sdodson Yeah, I understand that, my question more more along the lines of what we are calling the product_type for atomic-enterprise when installing origin...  is it still atomic-enterprise, or are we going to use a different name...  for openshift, the product is called openshift for both, but having an unsupported community product called atomic-enterprise just "feels" odd.